File: 01_Basico/03 - Lacos_Repeticao/Respsota_em_Python/questao14.py
while True:
    numero_tabuada = int(input("Digite o número para realizar sua tabuáda: "))
    numero_inicial = int(input("Digite o número pelo qual a tabuada vai começar: "))
    numero_final = int(input("Digite o número no qual a tabua vai encerrar: "))

    if numero_inicial > numero_final:
        print("Opção inválida, o número final deve ser maior que o inicial: ")
        continue
    else:
        for i in range(numero_inicial, numero_final + 1):
            print(f"A soma de {numero_tabuada} + {i} é: {numero_tabuada + i}")    

File: 01_Basico/03 - Lacos_Repeticao/Resposta_em_Kotlin/questao14.kt
fun main () {
    while (true){
    println("Digite um número para exibir sua tabuada")
    var numero_tabuada = readLine()!!.toInt()
    println("Digite o número inicial ")
    var numero_inicial = readLine()!!.toInt()
    println("Digite o número finl: ")
    var numero_final = readLine()!!.toInt()

    if (numero_inicial > numero_final) {
        println("Opção inválida, o número final deve ser maior que o inicial: ")
        continue
    }else{}
        for (i in numero_inicial .. numero_final){
            println("A soma de ${numero_tabuada} + ${i} é: ${numero_tabuada + i}")
        }
    break
    }
}